src/test/java/md/maib/CustomerControllerTest.java

```diff
@@ -0,0 +1,73 @@
+package md.maib;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import md.maib.entity.Customer;
+import md.maib.repository.CustomerRepository;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+import org.testcontainers.containers.PostgreSQLContainer;
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.hasSize;
+import java.util.List;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class CustomerControllerTest {
+
+    @LocalServerPort
+    private Integer port;
+
+    static PostgreSQLContainer<?> postgres = new PostgreSQLContainer<>(
+            "postgres:15-alpine"
+    );
+
+    @BeforeAll
+    static void beforeAll() {
+        postgres.start();
+    }
+
+    @AfterAll
+    static void afterAll() {
+        postgres.stop();
+    }
+
+    @DynamicPropertySource
+    static void configureProperties(DynamicPropertyRegistry registry) {
+        registry.add("spring.datasource.url", postgres::getJdbcUrl);
+        registry.add("spring.datasource.username", postgres::getUsername);
+        registry.add("spring.datasource.password", postgres::getPassword);
+    }
+
+    @Autowired
+    CustomerRepository customerRepository;
+
+    @BeforeEach
+    void setUp() {
+        RestAssured.baseURI = "http://localhost:" + port;
+        customerRepository.deleteAll();
+    }
+
+    @Test
+    void shouldGetAllCustomers() {
+        List<Customer> customers = List.of(
```

Comment:
use var

Reply:
Just an advice, new language feature, but it depends on code style in compony. 
We properly use **var**